src/Operation/DatabaseCommand.php

```diff
@@ -82,9 +82,8 @@ public function __construct(private string $databaseName, array|object $command,
      * Execute the operation.
      *
      * @see Executable::execute()
-     * @return Cursor
      */
-    public function execute(Server $server)
+    public function execute(Server $server): Cursor
```

Comment:
Should this be CursorInterface, or is the narrow return type sensible given we know exactly what the method does? I saw @GromNaN pointed something similar out elsewhere, so I'm not sure if it's the same issue (in which case there could also be others that were missed).

Reply:
While we technically know what return type is coming from the extension, using `CursorInterface` allows us to change the actual cursor implementation without causing a BC break. I've changed this and other types to `CursorInterface&Iterator`, which we can then replace with `CursorInterface` after PHPC-2447.

Reply:
This was changed to `CursorInterface`